components/gardencontent/profiles/manifests/manifest.yaml

```diff
@@ -48,13 +48,10 @@ defaults:
     caBundle: (( values.config.caBundle || ~~ ))
   kubernetes:
     versions:
-      - version: 1.17.1
-      - version: 1.16.4
-      - version: 1.16.3
-      - version: 1.15.7
-      - version: 1.15.6
+      - version: 1.17.2
+      - version: 1.16.6
+      - version: 1.15.9
```

Comment:
Recently were released v1.17.3, v1.16.7 and v1.15.10
* https://github.com/kubernetes/kubernetes/releases/tag/v1.17.3
* https://github.com/kubernetes/kubernetes/releases/tag/v1.16.7
* https://github.com/kubernetes/kubernetes/releases/tag/v1.15.10

Reply:
Thanks for the info, I'll update the commit.